caikit_computer_vision/data_model/__init__.py

```diff
@@ -11,3 +11,12 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
+
+# Local
+from . import image_classification
+from .image_classification import *
+
+from . import object_detection
+from .object_detection import *
+
+from . import tasks
```

Comment:
@gkumbhat any thoughts on keeping tasks here?

This better aligns with how caikit structures their interfaces, but in caikit-nlp, we define the generation task under our modules. I think this might be a bit cleaner, but we should be consistent. If you like it here, I'll move the generation task defs in caikit-nlp 😄

Reply:
Completely agree. Since these tasks would live in these domain specific libraries for a short time, it kinda doesn't matter much, but overall, in terms of when we add stuff in future, its an interesting pattern.

The reason I was thinking of these being in the module itself, was because pattern wise, the name of the folder inside the `module/` folder in domain specific lib like `caikit-computer-vision` is telling a user about a "task".. So I was thinking it would make sense for the module to have a `task.py` (or `<task-name>_task.py` file in there defining the task, so that whenever someone is looking at the `module/` they can identify the task based on the `*_task.py` file. But I do see that since its a "data model operation", it might make sense to have it in `data_model` folders. I think either works.

Reply:
Cool - in that case, I'd like to go with this structure, because it's easier to move to caikit; since there are no modules implemented in caikit, we'd have to aggregate the individual `<task>` defs to promote things if we kept things on the other pattern